modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,59 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-create-vms.adoc
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+[id='Importing_vmware_vm_or_template_{context}']
+= Importing a VMware virtual machine or template
 
-= Importing the VMware virtual machine or template
+You can import a VMware virtual machine or template using the virtual machine wizard.
 
 .Prerequisites
 
-* The VMware virtual machine is powered off.
+* You must create a VDDK image, push it to a private container repository, and add it to the `v2v-vmware` Configmap.
+* You must power off the VMware virtual machine before importing it.
 
 .Procedure
 
-. Click *Workloads* -> *Virtual Machines* from the side menu.
-. Click *Create Virtual Machine* and select *Create with Wizard*.
-. Perform the following procedure in the *Basic Settings* screen:
-.. Click *Provision Source* and select *Import*.
-.. Click *Provider* and select *VMware*.
-.. Click *vCenter Instance* and select *Connect to New Instance* or a saved vCenter instance.
-+
-* If you select *Connect to New Instance*, provide values in the following fields and click *Check and Save*:
+. In the {CNVProductName} web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create* and select *Import with Wizard*.
+. In the *General* screen, perform the following steps:
+.. Select *VMware* from the *Provider* list.
+.. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
 
-** *vCenter Hostname*
-** *vCenter User Name*
-** *vCenter Password*
-+
-The wizard connects to the vCenter instance and saves the credentials.
+.. Select a virtual machine or a template to import.
 +
-If you clear the *Save as New vCenter Instance* check box and click *Check*, the wizard connects to the vCenter instance without saving the credentials.
+The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
 
-* If you select a saved vCenter instance, the wizard connects to it by using the saved credentials.
+.. Select an existing flavor or *Custom* from the *Flavor* list.
+.. Select a *Workload Profile* and then click *Next*.
 
-.. Click *VM to import* and select a virtual machine or template.
-+
-The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
+. In the *Networking* screen, perform the following steps:
 
-.. Specify the *Memory (GB)* and number of *CPUs* if you use the default flavor, *Custom*.
+.. Click the Options menu {kebab} of a network interface and select *Edit*.
+.. Enter a valid network interface name.
 +
-Optionally, you can select an existing *Flavor*.
+The name can contain lower-case letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain upper-case letters, spaces, periods (`.`), or special characters.
 
```

Comment:
After 'Name' field there is a 'Model' field, that is not mentioned here. (It can contain "VirtIO for example).